build.gradle

```diff
@@ -441,6 +441,44 @@ task bundle(type: (isWindows()) ? Zip : Tar) {
     }
 }
 
+task bundleRestricted(type: Tar) {
+    dependsOn('crossDistPlatformLibs')
+    dependsOn('crossDist')
+    dependsOn('distSources')
+    def simpleOsName = HostManager.simpleOsName()
+    baseName = "kotlin-native-$simpleOsName-$konanVersionFull-restricted"
```

Comment:
I think it's better to have `kotlin-native-restricted-$simpleOsName-$konanVersionFull` instead. It's easier to support from the Gradle plugin side and IMHO more intuitive.